<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FocusDialog</class>
 <widget class="QDialog" name="FocusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢迎使用专注模式</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/assets/tomato.jpg</normaloff>:/assets/tomato.jpg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <layout class="QVBoxLayout" name="progressVBox">
     <item>
      <widget class="QWidget" name="progressContainer" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="quoteLayout">
     <item>
      <widget class="QLabel" name="sloganLabel">
       <property name="text">
        <string>名人名言</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="quoteSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>—— 作者</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout" stretch="0,0,1,0,0">
     <item>
      <widget class="QLabel" name="setTimeLabel">
       <property name="text">
        <string>专注分钟数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="timeSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="beginBtn">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
